tests/pipeline/misc/custom_reftax.nf.test

```diff
@@ -0,0 +1,53 @@
+nextflow_pipeline {
+
+    name "Test Workflow main.nf"
+    script "main.nf"
+    profile "test_reftaxcustom,docker"
```

Comment:
this pipeline uses multiple profiles and I had to place docker along with them in each test file. let me know if you can think of an avoiding this @Emiller88

Reply:
I've been down this road before 🤠 

```suggestion
    config "config/test_reftaxcustom.config"
```

Reply:
cool :sunglasses:

Reply:
i think we can do this for all the pipelines

Reply:
I spoke too fast....I went down this road too....declaring config here overrides the one set in nf-test.config and errors out as it cannot access nextflow.config now @Emiller88 thoughts?